client/src/Components/CarData.js

```diff
@@ -29,18 +29,14 @@ export default function CarData(props) {
     return (
       <div className="ui container">
         <Card.Group>
-          {/* <FuelLeft 
-          
-          /> */}
+          {console.log(data.car)}
```

Comment:
remove. Seems like this would fail eslint. Can you run eslint and fix any issues?